src/components/Homepage/home.css

```diff
@@ -0,0 +1,378 @@
+	background-image: url('../images/Component\ 12.png');
+	background-repeat: no-repeat;
+	position: absolute;
+	width: 237px;
+	height: 195px;
+	left: 909px;
+	top: 115px;
+	border-radius: 5px 5px 0px 0px;
+}
+
+.first-cards {
+	position: absolute;
+	 width: 237px;
+	height: 259px;
+	left: 30px;
+	top: 25px; 
+	border-radius: 5px;
+	background: #fff;
+	box-shadow: 0px 1px 4px rgba(0,  0,  0, 0.1);
+}
+
+.second-cards {
+	position: absolute;
+	 width: 237px;
+	height: 259px;
+	left: 30px;
+	top: 25px; 
+	border-radius: 5px;
+	background: #fff;
+	box-shadow: 0px 1px 4px rgba(0,  0,  0, 0.1);
+}
+
+.third-cards {
+	position: absolute;
+	 width: 237px;
+	height: 259px;
+	left: 30px;
+	top: -140px; 
+	border-radius: 5px;
+	background: #fff;
+	box-shadow: 0px 1px 4px rgba(0,  0,  0, 0.1);
+}
+
+.img-tag{
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	font-family: 'Open Sans', sans-serif;
+	font-size: 14px;
+	font-weight: normal;
+	font-style: normal;
+	line-height: 19px;
+	color: #444444;
+	margin-top: 14px;
+}
+
+.second-section-box {
+	background-image: url('../images/Component\ 12.png');
+	background-repeat: no-repeat;
+	position: absolute;
+	width: 237px;
+	height: 195px;
+	left: 744px;
+	top: 430px;
+	border-radius: 5px 5px 0px 0px;
+}
+
+.third-section-box {
+	background-image: url('../images/Component\ 12.png');
+	background-repeat: no-repeat;
+	position: absolute;
+	width: 237px;
+	height: 195px;
+	left: 1100px;
+	top: 600px;
+	border-radius: 5px 5px 0px 0px;
+}
+
+.lists1, .lists2, .lists3{
+	position: absolute;
+	width: 16px;
+	height: 14px;
+	left: -32px;
+	top: 4px;
+}
+
+#list1{
+	position: absolute;
+	width: 187px;
+	height: 22px;
+	left: 97px;
+	top: 473px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight: normal;
+	font-style: normal;
+	font-size: 16px;
+	line-height: 22px;
+}
+
+#list2{
+	position: absolute;
+	width: 446px;
+	height: 22px;
+	left: 97px;
+	top: 505px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight: normal;
+	font-style: normal;
+	font-size: 16px;
+	line-height: 22px;
+}
+
+#list3{
+	position: absolute;
+	width: 258px;
+	height: 22px;
+	left: 97px;
+	top: 536px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight: normal;
+	font-style: normal;
+	font-size: 16px;
+	line-height: 22px;
+}
+
+#get-started-btn{
+	position: absolute;
+	width: 177px;
+	height: 52px;
+	left: 60px;
+	top: 615px;
+	background: #356DFB;
+	border-radius: 30px;
+	border: none;
+	outline: none;
+}
+
+ a #get-started-btn{
+	text-decoration: none;
+	color: #E1E1EC;
+	font-family: 'Open Sans', sans-serif;
+	font-size: 16px;
+	line-height: 22px;
+	font-weight: normal;
+	font-style: normal;
+}
+
+footer{
+	position: absolute;
+	width: 208px;
+	height: 15px;
+	left: 631px;
+	top: 750px;
+	font-family: 'Open Sans', sans-serif;
+	font-size: 11px;
+	line-height: 15px;
+	font-weight: normal;
+	font-style: normal;
+}
+
+/********************* Media Querries *********************/
+/*
+@media screen and (max-width: 1100px){
```

Comment:
Dead code

Reply:
I got a lot of dead codes here boss,  you did not even go soft on me at all. But can you please  share more light on any of the dead code?